### ISSUE TITLE
-Wunused-variable in drivers/i2c/busses/i2c-designware-master.c

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#630]
Reported-by: kernelci.org bot <bot@kernelci.org>
```
>     drivers/i2c/busses/i2c-designware-master.c:658:6: warning: unused variable 'r' [-Wunused-variable]
>     1 warning generated.
```

**Comment by nathanchance** [comment]
Already fixed in the maintainer's tree: https://git.kernel.org/wsa/linux/c/f91b2ab0e0c5f7cb4b11012748a184b91a396f0f

**Comment by nathanchance** [comment]
Fixed in `next-20190807`.
